great_expectations/rule_based_profiler/rule_based_profiler.py

```diff
@@ -158,12 +154,20 @@ def _init_rules(
                     )
 
             # Instantiate builder attributes
-            domain_builder = self._init_domain_builder(rule_config, data_context)
-            parameter_builders = self._init_parameter_builders(
-                rule_config, data_context
+            domain_builder = RuleBasedProfiler._init_domain_builder(
+                domain_builder_config=rule_config["domain_builder"],
+                data_context=data_context,
+            )
+            parameter_builders = RuleBasedProfiler._init_parameter_builders(
+                parameter_builder_configs=rule_config.get("parameter_builders"),
```

Comment:
Could we make this change to ensure we have something to iterate over?
```suggestion
                parameter_builder_configs=rule_config.get("parameter_builders", []),
```

Reply:
Great catch -- of course! I am addressing it (slightly differently). Thanks!